Lib/test/test_math.py

```diff
@@ -2503,6 +2503,43 @@ def test_input_exceptions(self):
         self.assertRaises(TypeError, math.atan2, 1.0)
         self.assertRaises(TypeError, math.atan2, 1.0, 2.0, 3.0)
 
+    def test_exception_messages(self):
+        x = -1.1
+        with self.assertRaises(ValueError,
+                               msg=f"expected a nonnegative input, got {x}"):
```

Comment:
`msg=` is the error that gets raised if this assertion fails. To check the error message, you want `with self.assertRaisesRegex`.

Reply:
Thanks, fixed.